### PR TITLE
Hw-12

### PR DESCRIPTION
**Pull request by YaninaAQA**

### CODE REVIEW

File: tests/auth-order-creation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  const getOrderId = await request.get(`${serviceURL}${orderPath}/${orderId}`, {
+    headers: { Authorization: `Bearer ${token}` },
+  })
+  expect.soft(getOrderId.status()).toBe(StatusCodes.OK)
+})
+
+test('Authorize and delete order ID (without API client)', async ({ request }) => {
+  const loginDto = LoginDto.createLoginWithCorrectCredentials()
+  const auth = await request.post(`${serviceURL}${loginPath}`, { data: loginDto })
```
Comment (niamh-d) [inline tests/auth-order-creation.spec.ts]: The naming could be improved, e.g. authResponse

File: tests/api/api-client.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  }
+
+  async getOrderId(orderId: number): Promise<void> {
+    console.log(`Fetching order with ID ${orderId}`)
+    const response = await this.request.get(`${serviceURL}${orderPath}/${orderId}`, {
+      headers: {
+        Authorization: `Bearer ${this.jwt}`,
+      },
+    })
+    console.log('Order ID: ', response)
```
Comment (niamh-d) [inline tests/api/api-client.ts]: This logs the response, not the order/id only.

**Review by niamh-d — APPROVED** [review]

File: tests/auth-order-creation.spec.ts (hunk 1 above)
Comment (YaninaAQA) [reply tests/auth-order-creation.spec.ts]: Fixed

File: tests/api/api-client.ts (hunk 2 above)
Comment (YaninaAQA) [reply tests/api/api-client.ts]: Fixed

**Pull request merged**